app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -147,4 +140,9 @@ class KiwixMainActivity : CoreMainActivity() {
   override fun openSettingsActivity() {
     startActivityForResult(intent<KiwixSettingsActivity>(), REQUEST_PREFERENCES)
   }
+
+  override fun openPage(pageUrl: String, zimFilePath: String) {
+    val bundle = bundleOf("pageUrl" to pageUrl, "zimFileUri" to zimFilePath)
```

Comment:
if you are going to call the bundle a `bundle` then I'd rather the variable not exist. Also magic strings